ganga/GangaCore/Lib/Localhost/Localhost.py

```diff
@@ -183,11 +186,21 @@ def preparejob(self, jobconfig, master_input_sandbox):
         from GangaCore.Core.Sandbox.WNSandbox import PYTHON_DIR
         import inspect
 
-        fileutils = File( inspect.getsourcefile(GangaCore.Utility.files), subdir=PYTHON_DIR )
+        virtualization = job.virtualization
 
+        utilFiles= []
+        fileutils = File( inspect.getsourcefile(GangaCore.Utility.files), subdir=PYTHON_DIR )
+        utilFiles.append(fileutils)
+        if virtualization:
+            virtualizationutils = File( inspect.getsourcefile(GangaCore.Utility.Virtualization), subdir=PYTHON_DIR )
+            utilFiles.append(virtualizationutils)
+            if type(virtualization.image) is File:
+                utilFiles.append(virtualization.image)
```

Comment:
We should not allow an image to be a `File` object. It should always be a `GangaFile` object.